workflow_tests/utils/utils.ts

```diff
@@ -127,11 +160,12 @@ function setJobRunners(act, jobs, workflowPath) {
     return act;
 }
 
-function deepCopy(originalObject) {
+function deepCopy<T>(originalObject: T): T {
```

Comment:
How about this? More accurate with TS guidelines:
```suggestion
function deepCopy<TObject extends Record<string, unknown>>(originalObject: TObject): TObject {
```

Reply:
the signature in the definition is `any`, Record will restrict arrays for example

Reply:
How about `TObject extends Record<string, unknown> | unknown[]`?

Reply:
If you want to restrict it as much as possible, I'll need to check other use cases first, but since `stringify` and `lodash.cloneDeep` accept anything - I think there isn't much sense in doing so and we should keep the interface uniform